yarn.lock

```diff
@@ -2585,7 +2585,7 @@ __metadata:
     "@interlay/polkabtc-types": 0.7.4
     "@kiltprotocol/type-definitions": 0.1.8
     "@laminar/type-definitions": 0.3.1
-    "@phala/typedefs": 0.2.1
+    "@phala/typedefs": ^0.2.5
```

Comment:
```suggestion
    "@phala/typedefs": 0.2.5
```

We've had some issues with people using it this package and the dependencies (not this one), misbehaving. So apps is fine with this, however dependents may get some issues..

Reply:
changed